tests/API/OVAL/unittests/test_state_check_existence.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+result=`mktemp`
+
+set -e
+set -o pipefail
+
+cat > /tmp/foo.txt <<EOF
```

Comment:
@isimluk Unfortunately, a similar solution here is not possible, because it is not possible to use a relative path in //textfilecontent54_test/filepath element. The OVAL specification says regarding filepath: 
> The filepath element specifies the absolute path for a file on the machine. A directory cannot be specified as a filepath.